code/modules/atmospherics/machinery/components/binary_devices/volume_pump.dm

```diff
@@ -157,6 +166,16 @@
 		to_chat(user, "<span class='warning'>You cannot unwrench [src], turn it off first!</span>")
 		return FALSE
 
+/obj/machinery/atmospherics/components/binary/volume_pump/multitool_act(mob/living/user, obj/item/I)
+	if(!overclocked)
+		overclocked = TRUE
+		to_chat(user, "The pump makes a grinding noise and air starts to hiss out as you disable its pressure limits.")
+		desc += " Its warning light is on."
```

Comment:
This is not how you do desc. As it is, it would be stuck on "warning light is on" even if OC was disabled after the fact.

Consider using `examine` proc.

Reply:
![image](https://user-images.githubusercontent.com/19853115/52907452-09205680-321f-11e9-96de-7b2be50cdeca.png)

Its warning light is on.